.env.example

```diff
@@ -132,6 +132,7 @@ DEFAULT_MAIL_FROM_NAME="Republik"
 #  - republik to set interests based on belongings of a user
 #MAILCHIMP_URL=https://us9.api.mailchimp.com
 #MAILCHIMP_API_KEY=
+#MAILCHIMP_CAMPAIGN_CONFIGS='[{"key":"org/some-format","from_name":"Republik"}]'
```

Comment:
Describe the schema and possible keys here or link to this PR's description.

Reply:
Added a more verbose comment.